enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -36,10 +36,17 @@ const SdkContentWrapperInner = styled.div<
   }
 >`
   --mb-default-font-family: "${({ theme }) => getFontFamily(theme)}";
-  --mb-color-brand: ${color("brand")};
-  --mb-color-brand-alpha-04: ${alpha("brand", 0.04)};
-  --mb-color-brand-alpha-88: ${alpha("brand", 0.88)};
-  --mb-color-focus: ${color("focus")};
+  --mb-color-brand: ${({ theme }) => theme.fn.themeColor("brand")};
+  --mb-color-brand-alpha-04: ${({ theme }) =>
+    alpha(theme.fn.themeColor("brand"), 0.04)};
+  --mb-color-brand-alpha-88: ${({ theme }) =>
+    alpha(theme.fn.themeColor("brand"), 0.88)};
+  --mb-color-focus: ${({ theme }) => theme.fn.themeColor("focus")};
+  --mb-color-bg-white: ${({ theme }) => theme.fn.themeColor("bg-white")};
+  --mb-color-bg-black: ${({ theme }) => theme.fn.themeColor("bg-black")};
```

Comment:
note to future self: we should consider remapping `bg-black` and `bg-white` when we are tackling https://github.com/metabase/metabase/issues/42680 to remap colors to make sense for dark mode users